packages/tmux/build.sh

```diff
@@ -1,41 +1,21 @@
 TERMUX_PKG_HOMEPAGE=https://tmux.github.io/
 TERMUX_PKG_DESCRIPTION="Terminal multiplexer"
 TERMUX_PKG_LICENSE="ISC"
-TERMUX_PKG_MAINTAINER="@termux"
-_COMMIT=fbe6fe7f55cfc2a32f9cee4cb50502a53d3ce8bb
-_COMMIT_DATE=20230428
-TERMUX_PKG_VERSION=3.3a-p${_COMMIT_DATE}
-TERMUX_PKG_SRCURL=git+https://github.com/tmux/tmux
-TERMUX_PKG_SHA256=b61189533139bb84bdc0e96546a5420c183d7ba946a559e891d313c1c32d953d
-TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_MAINTAINER="Joshua Kahn @TomJo2000"
+TERMUX_PKG_VERSION=3.4
+TERMUX_PKG_SRCURL=https://github.com/tmux/tmux/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=ec7ddf021a0a1d3778862feb845fd0c02759dcdb37ba5380ba4e0038164f7187
 TERMUX_PKG_AUTO_UPDATE=false
 # Link against libandroid-support for wcwidth(), see https://github.com/termux/termux-packages/issues/224
 TERMUX_PKG_DEPENDS="ncurses, libevent, libandroid-support, libandroid-glob"
 # Set default TERM to screen-256color, see: https://raw.githubusercontent.com/tmux/tmux/3.3/CHANGES
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-static --with-TERM=screen-256color"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-static --with-TERM=screen-256color --enable-sixel"
 TERMUX_PKG_BUILD_IN_SRC=true
 
 TERMUX_PKG_CONFFILES="etc/tmux.conf etc/profile.d/tmux.sh"
 
-termux_step_post_get_source() {
-	git fetch --unshallow
-	git checkout $_COMMIT
-
-	local pdate="p$(git log -1 --format=%cs | sed 's/-//g')"
-	if [[ "$TERMUX_PKG_VERSION" != *"${pdate}" ]]; then
-		echo -n "ERROR: The version string \"$TERMUX_PKG_VERSION\" is"
-		echo -n " different from what is expected to be; should end"
-		echo " with \"${pdate}\"."
-		return 1
-	fi
-
-	local s=$(find . -type f ! -path '*/.git/*' -print0 | xargs -0 sha256sum | LC_ALL=C sort | sha256sum)
-	if [[ "${s}" != "${TERMUX_PKG_SHA256}  "* ]]; then
-		termux_error_exit "Checksum mismatch for source files."
-	fi
-}
-
 termux_step_pre_configure() {
+	CFLAGS='-Wno-implicit-function-declaration'
```

Comment:
`ntohl` and some other related functions are declared in `sys/endian.h`. It is better to include the header.

Reply:
Shouldn't the header be `arpa/inet.h` ?

Reply:
Emmm... In Android NDK, `arpa/inet.h` includes `netinet/in.h`, `netinet/in.h` includes `endian.h`, `endian.h` includes `sys/endian.h` and these functions are declared there.
Android NDK headers are not the same as GNU libcs, but as you mentioned, it is better to use a more generic one. So including `arpa/inet.h` as the man pages state is better.

Reply:
Sorry I just woke up, that'd just mean changing `CFLAGS` to,
```suggestion
	CFLAGS='-I???I'm blanking here???arpa/inet.h'
```

Reply:
I have sent a patch in upstream to fix the issue https://github.com/tmux/tmux/pull/3933. Please feel free to copy the required hunks for this package update.

Reply:
Used the patch of that PR and added it here.